libs/navigation/navigation.css

```diff
@@ -36,4 +36,8 @@ header.global-navigation, header.global-navigation.feds--dark {
   .feds-promo-link:hover {
     color: #136FF6;
```

Comment:
I'm guessing you can't use Milo variables in the standalone version, right? Even so, I'd argue it's valuable to define variables in this file as well, so that when changes are needed, updates are required in a single place. This, for example, matches `--link-hover-color`. L34 matches `--link-color`. Depending on how this standalone navigation is architected, you might want to use the same variable names or adapted ones.

Reply:
We will cover it in separate as it is not related to this issue.
If that works.